chat-plugins/scavengers.js

```diff
@@ -711,7 +711,6 @@ let commands = {
 			},
 
 			set: function (target, room, user) {
-				if (room.id !== 'scavengers') return this.errorReply("Scavenger games can only be created in the scavengers room.");
```

Comment:
Is removing this line intentional?

Reply:
afaik it's a meaningless check, since it requires a running scav game to already exist, which can only happen in valid rooms

Reply:
^ what whales said